### PR TITLE
Fix generated mod.rs to pass cargo fmt -- --check

### PR DESCRIPTION
**Pull request by hanabi1224**
Currently, the generated `mod.rs` does not pass `cargo fmt -- --check`, and this becomes a problem when ppl choose to include generated files as part of source code.

This PR tries to make `rustfmt` happy by adding an empty line after each 'pub mod' import.  It also adds `This file is generated by` line to make `mod.rs` consistent with other files.

This closes https://github.com/stepancheg/rust-protobuf/issues/621

### CODE REVIEW

File: protobuf-codegen/src/gen/mod_rs.rs [hunk 1]
```diff
@@ -3,9 +3,14 @@ use crate::gen::code_writer::CodeWriter;
 
 pub(crate) fn gen_mod_rs(mods: &[String]) -> compiler_plugin::GenResult {
     let v = CodeWriter::with_no_error(|w| {
-        w.comment(&format!("{}generated", "@"));
-        w.write_line("");
+        w.comment(&format!(
+            "This file is generated by {} {}. Do not edit",
+            env!("CARGO_PKG_NAME"),
```
Comment (stepancheg) [inline protobuf-codegen/src/gen/mod_rs.rs:8]: Code generator is not necessarily executed by cargo. See how similar code is inserted in other files.

File: protobuf-codegen/src/gen/mod_rs.rs [hunk 2]
```diff
@@ -3,9 +3,14 @@ use crate::gen::code_writer::CodeWriter;
...
+            env!("CARGO_PKG_VERSION")
+        ));
+        w.comment("@generated");
```
Comment (stepancheg) [inline protobuf-codegen/src/gen/mod_rs.rs:11]: That was

```
        w.comment(&format!("{}generated", "@"));
```

intentionally, so the generator is not considered to be generated.

**Review by stepancheg — CHANGES_REQUESTED** [review]
Two comments, and also can you split PRs into unrelated issues please?

File: protobuf-codegen/src/gen/mod_rs.rs (hunk 2 above)
Comment (hanabi1224) [reply protobuf-codegen/src/gen/mod_rs.rs]: Reverted

File: protobuf-codegen/src/gen/mod_rs.rs (hunk 1 above)
Comment (hanabi1224) [reply protobuf-codegen/src/gen/mod_rs.rs]: Removed

File: protobuf-codegen/src/gen/mod_rs.rs [hunk 3]
```diff
@@ -4,8 +4,8 @@ use crate::gen::code_writer::CodeWriter;
 pub(crate) fn gen_mod_rs(mods: &[String]) -> compiler_plugin::GenResult {
     let v = CodeWriter::with_no_error(|w| {
         w.comment(&format!("{}generated", "@"));
-        w.write_line("");
         for m in mods {
+            w.write_line("");
```
Comment (stepancheg) [inline protobuf-codegen/src/gen/mod_rs.rs:8]: Now it writes a line between each mod? Doesn't look right.

**Review by stepancheg — CHANGES_REQUESTED** [review]

File: protobuf-codegen/src/gen/mod_rs.rs (hunk 3 above)
Comment (hanabi1224) [reply protobuf-codegen/src/gen/mod_rs.rs:8]: The formatter won't sort mods when they are empty line delimited

Comment (stepancheg) [reply protobuf-codegen/src/gen/mod_rs.rs:8]: Wouldn't it be better to just sort mods?

Comment (hanabi1224) [reply protobuf-codegen/src/gen/mod_rs.rs]: Sure, if u prefer

**Comment by stepancheg** [comment]
Thanks!

**Pull request merged**